jetty-ee10/jetty-ee10-runner/src/main/java/org/eclipse/jetty/ee10/runner/Runner.java

```diff
@@ -111,26 +112,13 @@ public void addJars(Resource lib) throws IOException
             if (lib == null || !lib.exists())
                 throw new IllegalStateException("No such lib: " + lib);
 
-            List<String> list = lib.list();
-            if (list == null)
-                return;
-
-            for (String path : list)
+            for (Resource item: lib.list())
             {
-                if (".".equals(path) || "..".equals(path))
-                    continue;
-
-                Resource item = lib.resolve(path);
                 if (item.isDirectory())
                     addJars(item);
-                else
+                else if (FileID.isLibArchive(item.getFileName()))
                 {
-                    String lowerCasePath = path.toLowerCase(Locale.ENGLISH);
-                    if (lowerCasePath.endsWith(".jar") ||
-                        lowerCasePath.endsWith(".zip"))
-                    {
-                        _classpath.add(item.getURI());
-                    }
+                    _classpath.add(item.getURI());
```

Comment:
Remove braces for `else` as the `if` block does not have them.

Reply:
Done